src/queries/categories.js

```diff
@@ -66,3 +66,13 @@ export const GET_CATEGORIES_FILTER = gql`
     }
   }
 `;
+
+export const GET_CATEGORIES_FOR_PERSONALIZED_PUSH = gql`
```

Comment:
let's use the already existing `CATEGORIES_FILTER` with the same query attributes. we can extend the params for `tagList` next to `ids`. i will do it.